io.ente.auth.yml

```diff
@@ -0,0 +1,110 @@
+app-id: io.ente.auth
+runtime: org.gnome.Platform
```

Comment:
do you really need the GNOME runtime?

Reply:
I needed libwebkit2gtk support and one of the reference flutter app I saw was using similar runtime, just for this:
https://github.com/flathub/io.github.Predidit.Kazumi/blob/master/io.github.Predidit.Kazumi.yaml